static/lato.css

```diff
@@ -0,0 +1,93 @@
+/* Adapted from https://fonts.googleapis.com/css?family=Lato:400,400i,700,900 */
+
+/* non-latin (cyrillic, hungarian, serbian) */
+@font-face {
+    font-family: 'Lato';
+    font-style: italic;
+    font-weight: 400;
+    src: local('Lato Italic'), local('Lato-Italic'), url(fonts/lato/Lato-Italic.woff2) format('woff2');
+    unicode-range: U+0400-04FF, U+0500-052F, U+2DE0-2DFF, U+A640-A69F, U+1D00-1D7F;
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Lato Regular'), local('Lato-Regular'), url(fonts/lato/Lato-Regular.woff2) format('woff2');
+    unicode-range: U+0400-04FF, U+0500-052F, U+2DE0-2DFF, U+A640-A69F, U+1D00-1D7F;
+}
```

Comment:
There was some discussion in Slack about using `:lang()` pseudo selector when deciding which font files to load.  Why aren't we using it?

Reply:
Yeah, but the problem is that we could not do that currently because we are hard-coding the font-family values inside each Typography component. Maybe we can explore that possibility once we try a different CSS approach.

Reply:
One of the downsides of CSS-in-JS.